### PR TITLE
Updated WeatherHelper samples

### PR DESCRIPTION
**Pull request by MarkCiliaVincenti**
Updated samples to include the recently merged #783, including removing references to Summer Simmer Index which wasn't included, and also updated samples in some README.md files.

### CODE REVIEW

**Comment by MarkCiliaVincenti** [comment]
Ready for merge into master from my end.

**Comment by MarkCiliaVincenti** [comment]
Ready for merge into master from my end @krwq @joperezr

File: src/devices/Hts221/samples/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿# HTS221 - Capacitive digital sensor for relative humidity and temperature
+# HTS221 - Capacitive digital sensor for relative humidity and temperature
```
Comment (krwq) [inline src/devices/Hts221/samples/README.md:1]: I believe encoding changed in this file. Please revert to original

File: src/devices/Lps25h/samples/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿# LPS25H - Piezoresistive pressure and thermometer sensor
+# LPS25H - Piezoresistive pressure and thermometer sensor
```
Comment (krwq) [inline src/devices/Lps25h/samples/README.md:1]: revert encoding

File: src/devices/SenseHat/samples/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿# Sense HAT
+# Sense HAT
```
Comment (krwq) [inline src/devices/SenseHat/samples/README.md:1]: ditto

File: src/devices/Hts221/samples/README.md (hunk 1 above)
Comment (MarkCiliaVincenti) [reply src/devices/Hts221/samples/README.md]: Done, apologies these got inexplicably changed by Gitpod. Fixed.

File: src/devices/Lps25h/samples/README.md (hunk 2 above)
Comment (MarkCiliaVincenti) [reply src/devices/Lps25h/samples/README.md]: Done, apologies these got inexplicably changed by Gitpod. Fixed.

File: src/devices/SenseHat/samples/README.md (hunk 3 above)
Comment (MarkCiliaVincenti) [reply src/devices/SenseHat/samples/README.md]: Done, apologies these got inexplicably changed by Gitpod. Fixed.

**Comment by krwq** [comment]
Thanks @MarkCiliaVincenti!

**Comment by MarkCiliaVincenti** [comment]
> Thanks @MarkCiliaVincenti!

Thanks!

**Pull request merged**